__tests__/components/fields/array.js

```diff
@@ -0,0 +1,10 @@
+/*global describe, it, expect*/
+
+import { renderFieldsToHtml } from '../../../FormaticTestUtils';
```

Comment:
If we're sticking with `rollup` any want or thoughts of using the (resolve alias plugin](https://github.com/rollup/rollup-plugin-alias)

Reply:
Totally cool by me. I think I'll hold off for another PR for it. Maybe you're volunteering? 😄 If I understand, it means the tests use the rollup babel plugin? I hadn't thought about doing that and wasn't aware of that alias plugin either. In other projects, I've just used a symlink for tests: https://github.com/jdeal/qim/tree/master/test/node_modules.

Reply:
I'm either/or—no strong feeling as to how. But it might be nice to skip relative paths 🤷‍♂️